packages/eslint-config/src/index.mjs

```diff
@@ -0,0 +1,140 @@
+import eslint from '@eslint/js';
```

Comment:
This is the single source of truth for eslint config. ATM it has all projects in one file